x/ibc-transfer/keeper/relay.go

```diff
@@ -49,7 +51,7 @@ func (k Keeper) SendTransfer(
 
 	return k.createOutgoingPacket(
```

Comment:
thoughts on removing `createOutgoingPacket` and merging the code into `SendTransfer`? It would differ with the ics spec, but `createOutgoingPacket` does more than just construct the packet it also executes transfer logic. I think it's easier to understand `SendTransfer` as performing all send logic + constructing the packet needed to call the TAO Send logic

Reply:
+1. We can merge them and add a comment like:

```golang
SendTransfer
// ... 
// begin createOutgoingPacket logic


// end createOutgoingPacket

```